### ISSUE TITLE
ReactNative Version 0.46.+ Applications crashes on Android

### ISSUE THREAD

**Issue by riftshock** [invertase/react-native-firebase#285]
Good Day Friend, awesome package btw.

Basically title says it all, my app crashes using **RN version 0.46.**+ on **Android** without errors upon build and no errors on seen while having run react-native log-android.

I did tried doing fresh application using **RN version 0.42.3** which my teamates are currently using on their project and the app works fine.

Currently Developing on **macOS Sierra** Version 10.12.5
**Android Studio** 2.3.3
**React Native Version** 0.46.0
**RNFirebase Version** 2.0.4

Any Help would gladly appreciated. Godspeed Mate :)

**Comment by chrisbianca** [comment]
@riftshock I'm afraid that without any errors it's not possible to help you. We need to know whether this is a build error, or a runtime error, and have the associated error log before we can assist.

The logs from RN and from Android can be confusing, but you'll have to look through them and extract the error.

**Comment by riftshock** [comment]
@chrisbianca I agree, I'll try look deeper to the issue and I haven't tried IOS yet.. but maybe just in case someone had the same issue.. Thanks for the reply though, appreciate it =)

**Comment by riftshock** [comment]
@chrisbianca so I tried a fresh install again, a new project on **React-native both version 0.46.0 and 0.46.4** and firebase package works fine on Android Again.

Turns out there was a certain package that caused the issue of crashing of app  which is **react-native-device-info version 0.10.2** which is one I'm using on my current project.

Probably the reason the app crashes when I started a fresh new app is that I forgot to terminate the existing running packager on port 8081 for react native when I started building the new app, but I'm not entirely sure. My memory is not that accurate.

But anyways Thank You so much for your time. And cool package. Keep it up =) 
closing this now.
